spec/features/idv/steps/in_person/verify_step_spec.rb

```diff
@@ -186,4 +186,123 @@
       complete_verify_step(user)
     end
   end
+
+  context 'with in person verify info controller enabled ' do
+    let(:capture_secondary_id_enabled) { true }
+    let(:enrollment) { InPersonEnrollment.new(capture_secondary_id_enabled:) }
+    let(:user) { user_with_2fa }
+    let(:same_address_as_id) { true }
+    let(:double_address_verification) { true }
+
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_capture_secondary_id_enabled).
+        and_return(true)
+      allow(IdentityConfig.store).to receive(:in_person_verify_info_controller_enabled).
+        and_return(true)
+    end
+
+    it 'displays expected headers & data on /verify_info',
+       allow_browser_log: true do
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+      complete_prepare_step(user)
+      complete_location_step(user)
+      complete_state_id_step(
+        user, same_address_as_id: same_address_as_id,
+              double_address_verification: double_address_verification
+      )
+      click_idv_continue
+      complete_ssn_step(user)
+
+      # confirm url is /verify_info
+      expect(page).to have_current_path(idv_in_person_step_url(step: :verify_info))
+
+      # verify page
+      expect(page).to have_content(t('headings.verify'))
+
+      # confirm headers are on template
+      expect(page).to have_content(t('headings.state_id').tr(' ', ' '))
```

Comment:
```suggestion
      expect(page).to have_content(t('headings.state_id'))
```
Rather than do this, replace the weird space in `"State-issued ID"` (in `config/locales/headings/en.yml`, line 69) with a normal space.